contents/blog/clickhouse-vs-postgres.md

```diff
@@ -0,0 +1,186 @@
+---
+date: 2022-10-05
+title: "Postgres versus ClickHouse, Grapes versus Grapefruits"
```

Comment:
Trying to think of a compelling title that makes sense for this. A few options:

- Comparing ClickHouse and Postgres, different tools for different jobs
- Why ClickHouse and Postgres are different tools for different jobs

Reply:
I like the first one the most !